lib/vanagon/platform/windows.rb

```diff
@@ -119,8 +119,9 @@ def generate_msi_package(project)
         #   -gg             - Generate GUIDS now
         #   -dr             - Directory reference to root directories (cannot contains spaces e.g. -dr MyAppDirRef)
         #   -sreg           - Suppress registry harvesting.
-        #   -var <variable> - Substitute File/@Source="SourceDir" with a preprocessor or a wix variable
-        "cd $(tempdir); \"$$WIX/bin/heat.exe\" dir staging -v -ke -indent 2 -cg #{cg_name} -gg -dr #{dir_ref} -t wix/#{project.name}.filter.xslt -sreg -var var.StageDir -out wix/#{project.name}-harvest.wxs",
+        "cd $(tempdir); \"$$WIX/bin/heat.exe\" dir staging -v -ke -indent 2 -cg #{cg_name} -gg -dr #{dir_ref} -t wix/#{project.name}.filter.xslt -sreg -out wix/#{project.name}-harvest.wxs",
+        # candle individual files to generate the wxo files.
+        "cd $(tempdir); \"$$WIX/bin/candle.exe\"  -arch #{@architecture} -out wix/#{project.name}-harvest.wixobj   wix/#{project.name}-harvest.wxs  ",
```

Comment:
@melissa Had a look at the candle command in the Rakefile - The extra arguments appear to be to allow this command generalise all of the various input files (e.g. UI, Custom actions). This set of options is sufficient for the Harvest File only. 

I will probably leave this command as-is and then put in a more generalised command that works off a glob or array of filesnames (my preference) to be processed.

Would also prefer to keep the -out as it allows us to put the files in one place with the wixojb extension (don't know if it defaults to this, but could test it).